core/src/main/java/tech/pegasys/eth2signer/core/service/jsonrpc/SigningService.java

```diff
@@ -89,6 +91,8 @@ public String upcheck() {
 
   @JsonRpcMethod("Filecoin.WalletList")
   public List<String> filecoinWalletList() {
-    return fcKeyIdentifiers.list(KeyType.SECP256K1);
+    final List<String> secp = fcKeyIdentifiers.list(KeyType.SECP256K1);
+    final List<String> bls = fcKeyIdentifiers.list(KeyType.BLS);
+    return Stream.concat(secp.stream(), bls.stream()).collect(Collectors.toList());
```

Comment:
nit: list = new ArrayList<>(secp); list.addAll(bls) might be faster. but its just a nit pick.

Reply:
done